.env

```diff
@@ -1,2 +1,2 @@
-NEXT_PUBLIC_BASE_URL='https://staging-api.realdevsquad.com'
+NEXT_PUBLIC_BASE_URL='https://api.realdevsquad.com'
```

Comment:
Backend team will be hosting staging API soon, we don't need to change all the env files.
We want the development environment to use staging API.

We can revert this change.

Reply:
Okay! Will revert 👍🏾